maa-cli/src/config/task/condition.rs

```diff
@@ -138,6 +147,41 @@ pub fn remainder_of_day_mod(divisor: u32) -> u32 {
     day % divisor
 }
 
+fn game_date(datetime: DateTime<Utc>, client: &ClientType) -> Weekday {
```

Comment:
I would prefer if this function returns a `NaiveDate` instead of `Weekday` directly. Since `NaiveDate` implements the `Datelike` trait, we can then call `weekday()` on it.

Reply:
It seems that struct `NaiveDate`  calculates its weekday based on stored time, so creating a variable like "2024-02-14 Tue" appears unlikely.

Reply:
And since 2024-02-14, falls on a Wednesday, are you intending to create a weekday that does not match the date?